src/libraries/CashWrapper.sol

```diff
@@ -0,0 +1,45 @@
+pragma solidity ^0.4.13;
+
+import 'ROOT/trading/ICash.sol';
+import 'ROOT/Controlled.sol';
+import 'ROOT/Augur.sol';
+
+
+/**
+ * @title Provides modifiers which take care of Cash/ETH conversion
+ */
+contract CashWrapper is Controlled {
+    /**
+     * @dev Convert any ETH provided in the transaction into Cash before the function executes
+     */
+    modifier convertToCash() {
+        ethToCash();
+        _;
+        cashToETH();
+    }
+
+    /**
+     * @dev Withdraw any Cash held by the sender into ETH and transfer it to them after the function executes
+     */
+    modifier convertFromCash() {
+        _;
+        cashToETH();
+    }
+
+    function ethToCash() payable public returns (bool) {
```

Comment:
Is there a compelling reason for this being public?  Auditing is simpler if it is private (though only marginally).

Reply:
Originally I was going to test these outside of modifier use but it's not worth the tradeoff so I'll make em private